File: Projects/codechef/CUTBOARD.java
package codechef;

import java.util.Scanner;

class CUTBOARD {
    public static void main(String[] args) throws Exception {
        Scanner stdin = new Scanner(System.in);
        for (int i = stdin.nextInt(); i > 0; i--) {
            int rows = stdin.nextInt() - 1;
            int column = stdin.nextInt() - 1;
            System.out.println(rows * column);
        }
    }
}

File: Projects/codechef/CUTBOARD.kt
package codechef

import java.util.*

internal object CUTBOARD {
    @Throws(Exception::class)
    @JvmStatic
    fun main(args: Array<String>) {
        val stdin = Scanner(System.`in`)
        var T = stdin.nextInt()
        while (T-- > 0) {
            val N = stdin.nextInt()
            val M = stdin.nextInt()
            println((N - 1) * (M - 1))
        }
        stdin.close()
    }
}